languages.toml

```diff
@@ -2915,9 +2915,9 @@ scope = "source.just"
 file-types = [{ glob = "justfile" }, { glob = "Justfile" }, { glob = ".justfile" }, { glob = ".Justfile" }]
 injection-regex = "just"
 comment-token = "#"
-indent = { tab-width = 4, unit = "\t" }
-auto-format = true
-formatter = { command = "just", args = ["--dump"] }
+indent = { tab-width = 4, unit = " " }
```

Comment:
Is this change intentional? It will make the tab key indent by one space

Reply:
Oops intended to be four spaces. Checking...